Anagram.java

```diff
@@ -29,22 +28,63 @@ public static void main(String args[]) {
 
 	// Returns true if the two given strings are anagrams, false otherwise.
 	public static boolean isAnagram(String str1, String str2) {
-		// Replace the following statement with your code
-		return false;
+		str1=preProcess(str1);
```

Comment:
dont update the value of params.. 
usually it is better to declare a new variable